### PR TITLE
Microsite: Add new blog - Create, Manage, Explore

### PR DESCRIPTION
**Pull request by AlleyWei**
## Hey, I just made a Pull Request!

Add new blog: Create, Manage, Explore.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by dtuite** [comment]
Love this. This post will solve a real knowledge gap for the community.

File: microsite/blog/2021-05-20-adopting-backstage.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+**Pain points:**
+
+- When you’re this large, you’ve incorporated a lot of technology and processes, as well as a lot of partners, each with their own technologies and processes.
+- You’re focussed on “replacing cruft” and bringing discoverability and order to your vast ecosystem of components and tools.
+- Getting your tools adopted by your engineers while modernizing your tech stack and coordinating with your infra teams to improve your engineering practices across the whole company… it’s a lot.
+
+**Recommendation — Create, then manage, then explore:**
+
+- The fastest way to bring change to your ecosystem is to start at the beginning of the chain with Backstage Software Templates.
+- With every new component created with your templates, you’re establishing best practices and rewarding your developers with a streamlined experience focussed on their needs — all the while building up your new service catalog.
```
Comment (timbonicus) [inline microsite/blog/2021-05-20-adopting-backstage.md:130]:
```suggestion
- With every new component created with your templates, you’re establishing best practices and rewarding your developers with a streamlined experience focused on their needs — all the while building up your new service catalog.
```

**Review by timbonicus — APPROVED** [review]
Amazing article! 🔥 🔥

**Comment by AlleyWei** [comment]
Sorry, carryover DCO issues. Please approve - will note this for the future. =/

**Review by jhaals — APPROVED** [review]
Great post Austin!

**Pull request merged**